Windows CI breaks when the environment's LLVM doesn't match the version installed by scoop

Keeping up with this requires changing the explicit version installed by scoop (eg. `llvm@13.0.1`) whenever the Windows virtual environment updates with a new LLVM version: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md

Based on <https://github.com/Redfire75369/spiderfire/actions/runs/4497451094/jobs/7913055028>, MSVC and the version of LLVM 15.0.7 on the system definitely seem to be the issue. (Installed LLVM 14.0.6 with scoop)

```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include\yvals_core.h:807:1: error: static_assert failed "Error in C++ Standard Library usage.", err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2108:19: error: use of undeclared identifier '__builtin_elementwise_add_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2129:19: error: use of undeclared identifier '__builtin_elementwise_add_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2149:19: error: use of undeclared identifier '__builtin_elementwise_add_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2169:19: error: use of undeclared identifier '__builtin_elementwise_add_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2528:19: error: use of undeclared identifier '__builtin_elementwise_sub_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2548:19: error: use of undeclared identifier '__builtin_elementwise_sub_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2567:19: error: use of undeclared identifier '__builtin_elementwise_sub_sat', err: true
C:\Program Files\LLVM\lib\clang\15.0.7\include\emmintrin.h:2586:19: error: use of undeclared identifier '__builtin_elementwise_sub_sat', err: true
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -65,6 +65,14 @@ jobs:
         features: ["--features debugmozjs", ""]
         #target: [""]
         target: ["", "aarch64-uwp-windows-msvc", "x86_64-uwp-windows-msvc"]
+    env:
+        MOZTOOLS_PATH: 'C:\mozilla-build\msys\bin;C:\mozilla-build\bin'
+        AUTOCONF: "C:/mozilla-build/msys/local/bin/autoconf-2.13"
+        LINKER: "lld-link.exe"
+        CC: "clang-cl"
+        CXX: "clang-cl"
+        NATIVE_WIN32_PYTHON: "C:\\mozilla-build\\python2\\python.exe"
+        PYTHON3: "C:\\mozilla-build\\python3\\python3.exe"
     steps:
     - uses: actions/checkout@v3
     - uses: dtolnay/rust-toolchain@stable
@@ -78,10 +86,7 @@ jobs:
       run: |
         Start-BitsTransfer -Source https://ftp.mozilla.org/pub/mozilla/libraries/win32/MozillaBuildSetup-3.4.exe -Destination ./MozillaBuildSetup.exe
         .\MozillaBuildSetup.exe /S | Out-Null
-        iwr -useb get.scoop.sh -outfile 'install.ps1'
-        .\install.ps1 -RunAsAdmin
-        scoop install llvm@15.0.7 --global
-        echo "C:\ProgramData\scoop\shims;C:\Users\runneradmin\scoop\shims" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+        echo 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Tools\LLVM\bin' | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
     - name: Run sccache-cache
       uses: mozilla-actions/sccache-action@v0.0.3
       with:
@@ -89,30 +94,12 @@ jobs:
     - name: Build uwp
       if: contains(matrix.target, 'uwp')
       shell: cmd
-      env:
-        MOZTOOLS_PATH: 'C:\mozilla-build\msys\bin;C:\mozilla-build\bin'
-        AUTOCONF: "C:/mozilla-build/msys/local/bin/autoconf-2.13"
-        LINKER: "lld-link.exe"
-        CC: "clang-cl.exe"
-        CXX: "clang-cl.exe"
-        NATIVE_WIN32_PYTHON: "C:\\mozilla-build\\python2\\python.exe"
-        PYTHON3: "C:\\mozilla-build\\python3\\python3.exe"
-        LIBCLANG_PATH: "C:\\ProgramData\\scoop\\apps\\llvm\\current\\lib"
       run: |
         rustc --version --verbose
         cargo build --verbose ${{ matrix.features }} -Z build-std=std,panic_abort --target ${{ matrix.target }}
     - name: Build Windows
       if: contains(matrix.target, 'uwp') != true
       shell: cmd
-      env:
-        MOZTOOLS_PATH: 'C:\mozilla-build\msys\bin;C:\mozilla-build\bin'
-        AUTOCONF: "C:/mozilla-build/msys/local/bin/autoconf-2.13"
-        LINKER: "lld-link.exe"
-        CC: "clang-cl.exe"
-        CXX: "clang-cl.exe"
-        NATIVE_WIN32_PYTHON: "C:\\mozilla-build\\python2\\python.exe"
-        PYTHON3: "C:\\mozilla-build\\python3\\python3.exe"
-        LIBCLANG_PATH: "C:\\ProgramData\\scoop\\apps\\llvm\\current\\lib"
       run: |
         cargo test --verbose ${{ matrix.features }}
   Integrity:

